src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url("/fonts/Avenir-Heavy.ttf");
+}
```

```suggestion
@font-face {
  font-family: "Avenir";
  src: url("/fonts/Avenir-Book.ttf");
  font-weight: 300;
}

@font-face {
  font-family: "Avenir";
  src: url("/fonts/Avenir-Heavy.ttf");
  font-weight: 900;
}
```

That's how it should look like